moshi/src/main/java/com/squareup/moshi/Util.java

```diff
@@ -66,4 +67,17 @@ public static boolean hasNullable(Annotation[] annotations) {
     }
     return false;
   }
+
+  public static int[] doubleArray(int[] array) {
```

Comment:
Maybe `expandArray`? Only because a "double array" is a thing 😐

Reply:
Done.